decidim-admin/app/commands/decidim/admin/update_help_sections.rb

```diff
@@ -5,22 +5,36 @@ module Admin
     # This command gets called when a content block is updated from the admin
     # panel.
     class UpdateHelpSections < Decidim::Command
-      def initialize(form, organization)
+      def initialize(form, organization, user)
         @form = form
         @organization = organization
+        @user = user
       end
 
       def call
         return broadcast(:invalid) unless @form.valid?
 
         ActiveRecord::Base.transaction do
           @form.sections.each do |section|
-            ContextualHelpSection.set_content(@organization, section.id, section.content)
+            next unless content_has_changed?(section)
```

Comment:
For me this was not working correcty.

I updated only the "processes" section and this is what I see at the admin log:
![Admin log entries](https://user-images.githubusercontent.com/864340/170087300-e58f2643-14a8-4157-918a-24a9499be4f3.png)

Same thing first and second time when I did changes.

Is this on purpose?

Reply:
It's really strange it works perfectly for me

Reply:
looks like my "content_has_changed" function doesn't return what it's supposed to for the castellano content, but i'm not able to reproduce here

Reply:
> So maybe we just call it a Decidim feature?

With pleasure !